plugin/plugin.go

```diff
@@ -23,6 +26,25 @@ func New(policy string) validator.Plugin {
 
 func (p *plugin) Validate(ctx context.Context, req *validator.Request) error {
 
+	log.SetFormatter(&log.JSONFormatter{})
+	log.SetOutput(os.Stdout)
+
+	promotion := Promotion{
+		Build:       req.Build.Parent,
+		Environment: req.Build.Deploy,
+	}
+
+	// log each request for audit purposes
+	log.WithFields(log.Fields{
+		"extension":  "validation",
+		"user":       req.Build.Sender,
+		"event":      req.Build.Event,
+		"repository": req.Repo.Slug,
+		"branch":     req.Repo.Branch,
+		"commit":     req.Build.After,
+		"promotion":  &promotion,
+	}).Info(fmt.Sprintf("audit(%d)", time.Now().Unix()))
```

Comment:
**nit** Is their value to including the unix timestamp?   Splunk would include it.

Reply:
It's just a good practice for audit logs to include the exact time that the message was created to avoid discrepancies that can be caused by log shipping/buffer operations. This pattern is pretty common in operating system related logs (selinux, audit.log, etc) and low latency systems, but is totally optional here.

Do you think it would be better to leave an empty `msg` field instead?

Reply:
A "time" field should be automatically added to all log entries: https://github.com/sirupsen/logrus#entries

Reply:
Indeed, it's just a common practice to keep the timestamp on the message but as I said, def not needed. I'll keep the `msg` field empty instead.